addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -465,7 +465,7 @@
                                 <a  t-if="channel.can_upload"
                                     class="o_text_link o_wslides_js_slide_upload px-3 py-2"
                                     role="button"
-                                    aria-label="Upload Document"
+                                    aria-label="Add Content"
```

Comment:
A change of `aria-label`, I am curious why? :open_mouth:

Reply:
This is to avoid a warning of type `Elements with visible text labels do not have matching accessible names`. Indeed, inside of the anchor is an icon with `Add Content` in it :)